cl/search/api_serializers.py

```diff
@@ -49,6 +49,7 @@ class DocketSerializer(DynamicFieldsMixin, HyperlinkedModelSerializerWithId):
         view_name="court-detail",
         queryset=Court.objects.exclude(jurisdiction=Court.TESTING_COURT),
     )
+    court_id = serializers.CharField(read_only=True)
```

Comment:
Did you try using the ReadOnlyField? It's what HyperlinkedModelSerializerWithId uses?

Reply:
No, I didn't try to use that field, and Yes! the HyperlinkedModelSerializerWithId uses it. We can use the ReadOnlyField too because It simply returns the value of the field without modification.